ssc/api.py

```diff
@@ -43,8 +43,9 @@ def __init__(self):
 
     def _get_data_type(self, name):
         for key in self.keys():
+            key = key.decode('ascii')
```

Comment:
Why does this require ASCII? I think this should be taking `utf-8` if at all possible.

Reply:
Ah, that makes more sense